test/FixCode/fixits-apply-all.swift

```diff
@@ -2,7 +2,9 @@
 // The particular diagnostics used are not important.
 
 // RUN: %swift -parse -target %target-triple %s -fixit-all -emit-fixits-path %t.remap
-// RUN: c-arcmt-test %t.remap | arcmt-test -verify-transformed-files %s.result
+// RUN: c-arcmt-test %t.remap > %t.c-arcmt-test.out
+// RUN: arcmt-test -verify-transformed-files %s.result < %t.c-arcmt-test.out
+// RUN: sed -n 0~2p %t.c-arcmt-test.out | xargs rm -rf
```

Comment:
Right. So maybe, we should implement our frontend for this?
I will withdraw this change from the PR for now.